### PR TITLE
Fix inference for jittered resize

### PR DESCRIPTION
**Pull request by LukeWood**

### CODE REVIEW

**Review by tanzhenyu — CHANGES_REQUESTED** [review]
Please add unit test for it

**Comment by LukeWood** [comment]
> Please add unit test for it

Done

File: keras_cv/layers/preprocessing/jittered_resize.py [hunk 1]
```diff
@@ -131,8 +131,7 @@ def call(self, inputs, training=True):
         else:
             inputs = self._ensure_inputs_are_compute_dtype(inputs)
             inputs, meta_data = self._format_inputs(inputs)
-            output = inputs
-            output["images"] = self.inference_resizing(inputs["images"])
+            output = self.inference_resizing(inputs)
```
Comment (tanzhenyu) [inline keras_cv/layers/preprocessing/jittered_resize.py:134]: Make this private method instead

**Comment by LukeWood** [comment]
@tanzhenyu done

**Review by tanzhenyu — APPROVED** [review]

**Pull request merged**